doc/specs/friends.md

```diff
@@ -0,0 +1,67 @@
+# Authentication system
+
+## Metadata
+
+Author: flaming
+
+### Changelog
+
+**2015-08-02**
+
+* Initial version
+
+## Table of contents
+
+1. Motivation
+1. Business rules
+1. Use cases
+1. Data model
+
+## Motivation
+
+In a world with pseudonyms and nonstatic avatars rather than names and static faces (not counting facial burn victims, who account for only a small percent of the population), it helps to have a system by which users can tag other users as "worthy of remembering" in some sense, as well as record written notes pertaining to another user (e.g., "sent me a scarf for Secret Santa 2012! cool dude")
+
+## Business rules
```

Comment:
These business rules blur the line between what is a business rule and a use scenario. At least in the context of how the other specs have been written.